CMakeLists.txt

```diff
@@ -228,3 +245,18 @@ endif()
 if(BUILD_TESTING)
   add_subdirectory(example)
 endif()
+
+install(DIRECTORY ${CMAKE_SOURCE_DIR}/include/opentracing/
+        DESTINATION include/opentracing/
+        COMPONENT DEVEL)
+install(FILES ${CMAKE_BINARY_DIR}/output/libopentracing_mocktracer.so.${OPENTRACING_VERSION_STRING}
+        DESTINATION lib64/
+        COMPONENT DIST)
+install(FILES ${CMAKE_BINARY_DIR}/output/libopentracing.so.${OPENTRACING_VERSION_STRING}
+        DESTINATION lib64/
+        COMPONENT DIST)
```

Comment:
Instead these three installs, can the COMPONENT be added to the installs in the `if (BUILD_SHARED_LIBS)` block?

Also, the shared libraries won't be built if `-DBUILD_SHARED_LIBS=OFF` is specified.

Reply:
Installing through the existing BUILD_SHARED_LIBS call would include multiple unnecessary .so files (libopentracing.so, libopentracing.so.1, libopentracing.so.1.5.0) when only the last one is wanted and needed.